cmd/start/cmd.go

```diff
@@ -176,6 +166,27 @@ func enqueueNewBlocks(exportQueue types.HeightQueue, ctx *parser.Context) {
 	}
 }
 
+// mustGetLatestHeight will keep trying until it gets the latest height from RPC client, return 0 if retry count > 300
+func mustGetLatestHeight(ctx *parser.Context, retryCount int16) int64 {
+	if retryCount > 300 {
```

Comment:
300 is a bit high, I think we can go with 50 so then the total time of the retries for each block will take around 5 mins (if block time is approx. 6s)
```suggestion
	if retryCount > 50 {
```